src/sentry/conf/server.py

```diff
@@ -1791,6 +1791,11 @@ def get_sentry_sdk_config():
     KAFKA_INGEST_TRANSACTIONS: {"cluster": "default", "topic": KAFKA_INGEST_TRANSACTIONS},
 }
 
+# For Jira, only approved apps can use the access_email_addresses scope
+# This scope allows Sentry to use the email endpoint (https://developer.atlassian.com/cloud/jira/platform/rest/v3/#api-rest-api-3-user-email-get)
+# We use the email with Jira 2-way sync in order to match the user
+JIRA_USE_EMAIL_SCOPE = False
```

Comment:
Where/who does this approval?

Reply:
@markstory Jira/Atlassian does the approval process. You have to show that you are using the email address in a way they allow (in our case, linking users between systems).